frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -150,7 +150,9 @@ const CATEGORY_FILTER = createMockParameter({
   name: "Category",
   slug: "category",
 });
+const NUMBER_FILTER_ID = "number-hex";
 const DATE_FILTER_ID = "date-hex";
+const UNIT_OF_TIME_FILTER_ID = "unit-of-time-hex";
```

Comment:
Just a curiosity question: what does the `-hex` suffixes mean in these cases?

Reply:
Normally the parameter (filter) id will look something like `fc2cd1be` hence the `hex`. But I don't want to have to use the actual hex since it doesn't provide any information.

Reply:
that was indeed a mistake. Thanks!
#### After 
![Screenshot 2024-08-01 at 9 47 59 PM](https://github.com/user-attachments/assets/22e24612-5bb4-4e86-8e73-cf39f00f8d1c)


#### Before
![Screenshot 2024-08-01 at 9 35 01 PM](https://github.com/user-attachments/assets/b4c24b08-a5d7-4c5f-9f95-a4d1774c0b3d)

Reply:
Fixed in https://github.com/metabase/metabase/pull/46365/commits/2633e0f2501900fa9af0b60e51342ff17bedcb02